Desktop/property-pro-app/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import "./footer.css";
+
+export default function Footer() {
+  return (
+    <div className="footer">
+      <div className="footer-first-child">
+        <span className="about">ABOUT US</span>
+        <span className="contact">CONTACT US</span>
+      </div>
+      <div className="footer-second-child">
+        <span className="copyright"> COPYRIGHT</span>
+        <span className="preference"> COOKIE PREFERENCE</span>
+      </div>
```

Comment:
These footer links are supposed to be clickable right? I thought they are supposed to use the `<Link>` tag.